JAICore/jaicore-ml/src/main/java/ai/libs/jaicore/ml/core/dataset/serialization/ArffDatasetAdapter.java

```diff
@@ -164,14 +169,21 @@ protected static KVStore parseRelation(final String line) {
 		return metaData;
 	}
 
+	/**
+	 * parsing Arff string that defines an Attribute to an IAttribute

```

Comment:
Parses an attribute definition of an arff file. General format: 
@ATTRIBUTE <attribute_name> <attribute_type>